packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -212,18 +212,22 @@ class SessionsController {
       address
     );
 
-    // Get a new session signature.
+    // // Get a new session signature.
+    // if (!hasAuthenticatedSession) {
```

Comment:
Can we add something searchable here -- or alternatively, gate this block with the session keys flag?